### ISSUE TITLE
In reflection mode, the package is always "default"

### ISSUE THREAD

**Issue by simplesteph** [ktr0731/evans#136]
When specifying a proto file, we can select the appropriate package, but not using the reflection mode

**Comment by panzerdev** [comment]
I looked into this a bit and found following...

https://github.com/ktr0731/evans/blob/587ed17a4fdd8d6679ac34b6b8f10a102cf7d3d3/adapter/grpc/reflection.go#L50

Here in the response there is `*desc.ServiceDescriptor`

```type ServiceDescriptor struct {
	proto      *dpb.ServiceDescriptorProto
	file       *FileDescriptor
	methods    []*MethodDescriptor
	fqn        string
	sourceInfo *dpb.SourceCodeInfo_Location
}
```

And there the Package can be retrived `svc.GetFile().GetPackage()`

So maybe it's possible to show the right package even in reflection mode.

**Comment by ktr0731** [comment]
Thank you @panzerdev. It's a great function I hadn't known!
I reopen this issue to implement such a feature.

**Comment by ktr0731** [comment]
fixed in #158
